sdk/cognitiveservices/cognitiveservices-luis-runtime/src/lUISRuntimeClient.ts

```diff
@@ -1,8 +1,4 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- *
```

Comment:
Can you check why the copyright went missing?
There is a flag `license-header` which you might want to use when running the code generator

Reply:
I'll regenerate using this flag.

Reply:
`license-header` flag not working as autorest says can't convert object to string.
Does the order of the flags matter?

Reply:
I believe the order matters, but not sure.
@michaeljqzq Can you share how you used the code generator for management libraries to be able to get the Copyright info?

Reply:
Try passing `--license-header=MICROSOFT_MIT_NO_VERSION` instead of just `license-header`